desktop/core/src/desktop/js/parse/hplsqlStatementsParser.test.js

```diff
@@ -0,0 +1,58 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import hplsqlStatementsParser from './hplsqlStatementsParser';
+import { testParser, runSharedStatementsParserTests } from './sharedStatementsParserTests.ts';
+
+describe('hplsqlStatementsParser.js', () => {
+  runSharedStatementsParserTests(hplsqlStatementsParser);
```

Comment:
You should remove the extracted tests from sqlStatementsParser.test.js and run this there as well.

Reply:
done